client/src/components/ResultsDisplay/ResultsDisplay.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from "react";
+import { Box, Link, Typography } from "@material-ui/core";
+import BackButton from '../BackButton/BackButton';
+import DownloadButton from '../DownloadButton/DownloadButton';
+import FactList from '../FactList/FactList';
+import HealthCard from '../HealthCard/HealthCard';
+import JsPdf from "jspdf";
+import html2canvas from "html2canvas";
+
+const ResultsDisplay = () => {
+  const [fileDownload, setFileDownload] = useState(null);
+
+  const ref = React.createRef();
+
+  const handlePdfDownload = () => {
+    const source = ref || this.ref;
+    const targetComponent = source.current || source;
+    html2canvas(targetComponent, {
```

Comment:
This may be means for another ticket but could we create a separate React Component to use as the source for PDF generation? This way we could:
1. Remove the "BACK TO SEARCH" text on the top left of the page
2. Remove the eye symbol next to DOB, and enforce that DOB is always visible in the PDF version (having it hidden on a static page seems to be a bit odd)

Maybe the best way to do this would be to break up these parts of the current display page into smaller sub-components?